src/tests/ftest/pool/create_capacity.py

```diff
@@ -55,18 +72,15 @@ def test_create_pool_quantity(self):
         except ServerFailed as error:
             self.fail(error)
 
-        duration = float(time.time()) - start
-        self.assertLessEqual(
-            duration, 120,
-            "DAOS not ready to accept requests with in 2 minutes")
-
-        self.dmg.timeout = 360
+        if float(time.time()) - start > 120:
+            self.fail("DAOS not ready to accept requests with in 2 minutes after restart")
```

Comment:
pre-existing, but...
```suggestion
            self.fail("DAOS not ready to accept requests within 2 minutes after restart")
```

Reply:
Done.